rubyau/constitution.md

```diff
@@ -1,468 +1,779 @@
+  
+#### 67 Management of funds
 
-## 33. Funds
+1. The Association must open an account with a financial institution from which all expenditure of the Association is made and into which all of the Association's revenue is deposited.
 
-1. The Treasurer of the Association must—
-  1. collect and receive all moneys due to the Association and make all payments  authorised by the Association; and 
-  2. keep correct accounts and books showing the financial affairs of the  Association with full details of all receipts and expenditure connected with the  activities of the Association. 
+2. Subject to any restrictions imposed by a general meeting of the Association, the Committee may approve expenditure on behalf of the Association.
 
-2. All cheques, drafts, bills of exchange, promissory notes and other negotiable  instruments must be signed by two members of the committee. 
+3. The Committee may authorise the Treasurer to expend funds on behalf of the Association (including by electronic funds transfer) up to a specified limit without requiring approval from the Committee for each item on which the funds are expended.
 
-3. The funds of the Association shall be derived from entrance fees, annual  subscriptions, donations and such other sources as the committee determines. 
+4. All cheques, drafts, bills of exchange, promissory notes and other negotiable instruments must be signed by 2 committee members.
 
-## 34. Notice to members
+5. All funds of the Association must be deposited into the financial account of the Association no later than 5 working days after receipt.
 
-Except for the requirement in ‘Notice of general meetings’, any notice that is required to be given to a member, by or on behalf of the Association, under these Rules may be given by—
+6. With the approval of the Committee, the Treasurer may maintain a cash float provided that all money paid from or paid into the float is accurately recorded at the time of the transaction.
 
-1. delivering the notice to the member personally; or 
-2. sending it by prepaid post addressed to the member at that member's address  shown in the register of members; or 
-3. facsimile transmission, if the member has requested that the notice be given to  him or her in this manner; or 
-4. electronic transmission. 
+#### 68 Financial records
 
-## 35. Winding up
+1. The Association must keep financial records that—
+  1. correctly record and explain its transactions, financial position and performance; and
+  2. enable financial statements to be prepared as required by the Act.
 
-In the event of the winding up or the cancellation of the incorporation of the Association, the assets of the Association must be disposed of in accordance with the provisions of the Act.
+2. The Association must retain the financial records for 7 years after the transactions covered by the records are completed.
 
-## 36. Custody and inspection of books and records
+3. The Treasurer must keep in
```

Comment:
There is no note after Rule 18 and I think Rule 18's explanation is clear enough on this. It may do just to mention Rule 18 here once more.